### ISSUE TITLE
Tests have started to fail

### ISSUE THREAD

**Issue by andk** [kaz-utashiro/optex-pingu#1]
Sample fail report: http://www.cpantesters.org/cpan/report/dff091a4-0f78-11ed-b078-c423c0b3760d

I've reproduced the fail report by picking a perl that had a PASS report and then upgrading File::Share to INGY/File-Share-0.26.tar.gz

After that upgrade the test failed.

The two reports before/after: http://www.cpantesters.org/cpan/report/1a64e0c8-0f89-11ed-ad5b-8ac72b549893 http://www.cpantesters.org/cpan/report/50924d70-0f89-11ed-8503-7bca2b549893

@ingydotnet , can you have a look?

**Comment by ingydotnet** [comment]
@andk, definitely.

If I can't figure it out, I'll release 0.27 with the previous code.

**Comment by ingydotnet** [comment]
It's strange to see that https://www.cpantesters.org/distro/F/File-Share.html?oncpan=1&distmat=1&version=0.26&grade=2 is 400+ PASS and 0 FAIL.

The code seems to be failing here:
```
    my $path = $INC{$inc} || '';
    $path =~ s/$inc$//;
    $path = Cwd::realpath( File::Spec->catfile($path, '..') );

    if ($path =~ m<^(.*?)[\/\\]blib\b> and
```
with
```
Error:  Modification of a read-only value attempted at /home/sand/src/perl/repoperls/installed-perls/host/k93msid/v5.36.0/fc43/lib/site_perl/5.36.0/File/Share.pm line 32.
```
Why would `$path` be readonly? And if it is, what's a simple way to make it not?

**Comment by ingydotnet** [comment]
@andk out of curiosity, how did you come across this? This module is super new, v0.02 and just a few commits old.

**Comment by ingydotnet** [comment]
Got some help from ANEI, LEONT and TINITA. Showed me it was `$_` that was readonly.

0.27 in a few minutes.

**Comment by ingydotnet** [comment]
File-Share-0.27.tar.gz Released
Should fix things.

**Comment by kaz-utashiro** [comment]
Thank you all for solving problem.
I released a new version requiring File::Share@0.27.

By the way, have you ever tried pingu? :)
```
optex -Mpingu --pingu ping -c15 localhost
```

**Comment by kaz-utashiro** [comment]
Looks fine.  I close this issue.
http://matrix.cpantesters.org/?dist=App-optex-pingu+0.03
